src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -7,6 +7,7 @@
 #include <string.h>
 
 #include "locale.hpp"
+#include "calendarData.hpp"
```

Comment:
(minor IMO) I don't see a reason to use .h at this time. If we plan on supporting C consumers then .h is probably best, but if we ever add C++ stuff to that header (templates, include other .hpp, etc), then we should start with .hpp or have both .h and .hpp files.